### PR TITLE
[DOC][NFC] Fix change log indentation

### PR DESCRIPTION
**Pull request by bghimireamd**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,540 +3,540 @@
...
+- Softmax grid-size selection
+- debug prints of kernel launch parameters.
 ### Removed
-	- GEMM interface from the MIOpen API
+- GEMM interface from the MIOpen API
 
 ## MIOpen 1.0.0 for ROCm 1.0.0
 ### Added
-	-Initial release 
+-Initial release
```
Comment (saadrahim) [inline CHANGELOG.md]: spacing

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,540 +3,540 @@
 
 Full documentation for MIOpen is available at [https://rocmsoftwareplatform.github.io/MIOpen/doc/html/](https://rocmsoftwareplatform.github.io/MIOpen/doc/html/)
 
-## (Unreleased) MIOpen-2.18.0 for ROCm 5.5.0
+## (Unreleased) MIOpen-2.19.0 for ROCm 5.5.0
 ### Added
-	- ROCm 5.5 support for GFX1101 (Navi32)
+- ROCm 5.5 support for GFX1101 (Navi32)
```
Comment (saadrahim) [inline CHANGELOG.md]: gfx is usually lowercase, gfx1101

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,540 +3,540 @@
...
-	- Fix 3d convolution Host API bug
-	- [HOTFIX][MI200][FP16] Disabled ConvHipImplicitGemmBwdXdlops when FP16_ALT is required. 
+- Fix 3d convolution Host API bug
+- [HOTFIX][MI200][FP16] Disabled ConvHipImplicitGemmBwdXdlops when FP16_ALT is required. 
 
 ## MIOpen 2.16.0 for ROCm 5.1.1
 ### Notes
-	- This release includes enhanced support for MI210 and MI250 and various other improvements.
-	- This release consists of various bug fixes and performance improvements
+- This release includes enhanced support for MI210 and MI250 and various other improvements.
```
Comment (saadrahim) [inline CHANGELOG.md:18]: along with various other improvements, too many ands

**Review by saadrahim — DISMISSED** [review]
Some minor fixes, feel free to merge after addressing them.

**Review by saadrahim — APPROVED** [review]

**Review by junliume — APPROVED** [review]

**Pull request merged**